migrations/20170802152927-add_imc_profile_migration.js

```diff
@@ -1,27 +1,77 @@
 module.exports = {
   up: function (qi, Sequelize) {
-    return qi.addColumn('Profiles', 'botAccess', {
-      type: Sequelize.ENUM('r', 'rw'),
-      defaultValue: 'r',
+    qi.sequelize.query('SELECT * FROM "Profiles" LIMIT 1',
```

Comment:
I tried this originally, but I forgot the "return" and it didnt work; Ive updated the PR